holmes/main.py

```diff
@@ -262,6 +268,10 @@ def alertmanager(
     except Exception as e:
         logging.error(f"Failed to fetch issues from alertmanager", exc_info=e)
         return
+    
+    if alertmanager_limit is not None:
+        console.print(f"[bold yellow]Limiting to {alertmanager_limit}/{len(issues)} issues.[/bold yellow]")
+        issues = issues[:alertmanager_limit]
```

Comment:
is this an issue if the limit is bigger than the length of the array?

Reply:
No, it's fine. 

```python
>>> [0,1,2][:10]
[0, 1, 2]
```